IFC4x3/Properties/t/TrackEdge_18qZEZZgH0YeAVlHW1LDUp/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TrackEdge_18qZEZZgH0YeAVlHW1LDUp" Name="TrackEdge" UniqueId="48d233a3-8ea4-408a-829f-bd180154d7b3" PrimaryDataType="IfcIdentifier" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="TrackEdge" UniqueId="48d233a3-8ea4-408a-829f-bd180154d7b3" PrimaryDataType="IfcIdentifier" />
```

Comment:
Need to completely remove

Reply:
Done. Commit : c0f5835a24d8f1939a80ada93c3b1cb04772002d